src/components/back-to-top/BackToTop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+class BackToTop {
+  constructor() {
+    this._settings = {
+      selector: BackToTop.selector
+    }
+
+    //Get the button
+    const button = document.querySelector(this._settings.selector)
+
+    window.addEventListener('scroll', function(e) {
+      BackToTop.revealButton(button);
+    });
+
+    button.addEventListener('click', function(e) {
+      BackToTop.scrollTop()
+    });
+  }
+}
+
+// When the user scrolls down 20px from the top of the document, show the button
+BackToTop.scrollTop = function () {
+  document.body.scrollTop = 0;
+  document.documentElement.scrollTop = 0;
+}
+
```

Comment:
@yone920 is there an additional function you can add to have the button stop at the top of the footer?